IS7024/Pages/HollywoodBowl.cshtml.cs

```diff
@@ -30,33 +30,28 @@ public void OnGet()
                 string itstime = "~" + realtime + ":" + vars[1] + " PM";
                 ViewData["sun"] = itstime;
 
-
                 string jsonString = webClient.DownloadString("https://api.songkick.com/api/3.0/venues/7211/calendar.json?apikey=Y77nbW56nkfpIpw9");
-                JSchema schema = JSchema.Parse(System.IO.File.ReadAllText("SongkickSchema.json"));
+                JSchema schema = JSchema.Parse(System.IO.File.ReadAllText("SongkickSchemaAll.json"));
                 JObject jsonObject = JObject.Parse(jsonString);
                 IList<string> validationEvents = new List<string>();
-                //if (jsonObject.IsValid(schema, out validationEvents))
```

Comment:
I uncommented this part now that there was a Schema that could be used (SongkickSchemaAll)

Reply:
Good.  Commented out code should not be in version control.
1) If you are curious what used to be there, diff commits to see history.
2) If it is an experimental feature, use a feature branch to keep it separate from master.